### ISSUE TITLE
OpenSCAP emits many warnings while scanning Common profile for Fedora

### ISSUE THREAD

**Issue by jan-cerny** [OpenSCAP/openscap#716]
When I run a scan using SSG common profile for Fedora, I can see some warning messages at the end of the output:

I run this command:
```oscap xccdf eval --results-arf arf.xml --profile xccdf_org.ssgproject.content_profile_common --report report.html /usr/share/xml/scap/ssg/content/ssg-fedora-ds.xml```

And the output is:
```
[snip]
...
Title   Enable the Audit Daemon
Rule    xccdf_org.ssgproject.content_rule_service_auditd_enabled
Result  pass

W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
W: oscap: rbt_str_add: non-zero return code
...
[snip]
```

I'm using OpenSCAP 1.2.14 on Fedora 24 Workstation:
```
$ rpm -q openscap
openscap-1.2.14-1.fc24.x86_64
```

**Comment by jan-cerny** [comment]
Experienced also on F25.

**Comment by jan-cerny** [comment]
Fixed by #729
